src/redteam/graders.ts

```diff
@@ -77,6 +77,7 @@ export const GRADERS: Record<RedteamAssertionTypes, RedteamGraderBase> = {
   'promptfoo:redteam:harmful:privacy': new HarmfulPrivacyGrader(),
   'promptfoo:redteam:harmful:profanity': new ProfanityGrader(),
   'promptfoo:redteam:harmful:radicalization': new RadicalizationGrader(),
+  'promptfoo:redteam:harmful:rag-full-document-exfiltration': new CopyrightViolationGrader(),
```

Comment:
```suggestion
  'promptfoo:redteam:rag-document-exfiltration': new CopyrightViolationGrader(),
```

Where is the grader? How are you thinking about grading this?

Reply:
I think you should drop the `harmful:` prefix